ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -282,6 +282,37 @@ FAN_SS_DRSM8X:
   - SS
   - DRSM8X
 
+FAN_EDPM_ZHM_ZTM_LPS:
+  id: "16288085548559433728"
+  description: "Exhaust fan with sensors (temperature, humidity and low pressure)."
+  is_canonical: true
+  implements:
+  - FAN
+  - EDPM
+  - ZHM
+  - ZTM
+  uses:
+  - run_command
+  - run_status
+  - control_mode
+  - low_differential_pressure_alarm
```

Comment:
these should be opt_uses as they are uncommon.

Reply:
they have been defined under opt_uses